### PR TITLE
Mobile styles

### PR DESCRIPTION
**Pull request by gregpriday**
Resolves GH-269

### CODE REVIEW

File: inc/renderer.php [hunk 1]
```diff
@@ -133,101 +137,105 @@ public function generate_css( $post_id, $panels_data = false, $layout_data = fal
...
-					'margin-bottom' => apply_filters( 'siteorigin_panels_css_row_margin_bottom', $panels_margin_bottom . 'px', $row, $ri, $panels_data, $post_id )
+					'margin-bottom' => $panels_margin_bottom
 				) );
 			}
 
 			$collapse_order = ! empty( $row['style']['collapse_order'] ) ? $row['style']['collapse_order'] : ( ! is_rtl() ? 'left-top' : 'right-top' );
 
-			if ( $settings['responsive'] ) {
-
+			if ( $settings['responsive'] && ! empty( empty( $row['style']['collapse_behaviour'] ) ) ) {
```
Comment (braamgenis) [inline inc/renderer.php]: This is pretty hard to read and seems unnecessary. `! empty( empty( $var ) )` should be the same as just `empty( $var )`?

File: inc/renderer.php [hunk 2]
```diff
@@ -133,101 +137,105 @@ public function generate_css( $post_id, $panels_data = false, $layout_data = fal
...
-							$remove_bottom_margin .= 'last-child(' . ( count( $row['cells'] ) % 2 == 0 ? '-n+2' : '1' ) . ')';
-						} else {
-							$remove_bottom_margin .= 'child(-n+2)';
-						}
+					$css->add_cell_css( $post_id, $ri, false, '', array(
+						'-ms-flex'      => '0 1 50%',
+						'-webkit-flex'  => '0 1 50%',
+						'flex'          => '0 1 50%',
+						'margin-right'  => '0',
+						'margin-bottom' => $panels_margin_bottom,
```
Comment (braamgenis) [inline inc/renderer.php]: As far as I can tell, this is the only line that changed to remove the `'px` unit. That correct? Not sure why the diff marked so much as changed around this line. 😕

File: inc/renderer.php (hunk 1 above)
Comment (gregpriday) [reply inc/renderer.php]: Aaah yeah, that's a copy paste typo. Will fix it up.

**Comment by gregpriday** [comment]
Yeah, the diff got a bit crazy because I combined 2 if statements that shouldn't have been separate. I'll fix up the double empty.

**Pull request merged**